CMake/mlpack.cmake

```diff
@@ -630,29 +634,29 @@ macro(fetch_mlpack COMPILE_OPENBLAS)
     get_deps(https://www.ensmallen.org/files/ensmallen-${ENSMALLEN_FETCH_VERSION}.tar.gz ensmallen ensmallen-${ENSMALLEN_FETCH_VERSION}.tar.gz)
     set(ENSMALLEN_INCLUDE_DIR ${GENERIC_INCLUDE_DIR})
     find_ensmallen(${CMAKE_BINARY_DIR})
-    if (ENSMALLEN_FOUND)
-      set(MLPACK_INCLUDE_DIRS ${MLPACK_INCLUDE_DIRS} ${ENSMALLEN_INCLUDE_DIR})
-    endif()
+  endif()
+  if (ENSMALLEN_FOUND)
+    set(MLPACK_INCLUDE_DIRS ${MLPACK_INCLUDE_DIRS} ${ENSMALLEN_INCLUDE_DIR})
   endif()
 
   find_cereal(${CMAKE_BINARY_DIR})
   if (NOT CEREAL_FOUND)
     get_deps(https://github.com/USCiLab/cereal/archive/refs/tags/v${CEREAL_FETCH_VERSION}.tar.gz cereal cereal-${CEREAL_FETCH_VERSION}.tar.gz)
     set(CEREAL_INCLUDE_DIR ${GENERIC_INCLUDE_DIR})
     find_cereal(${CMAKE_BINARY_DIR})
-    if (CEREAL_FOUND)
-      set(MLPACK_INCLUDE_DIRS ${MLPACK_INCLUDE_DIRS} ${CEREAL_INCLUDE_DIR})
-    endif()
+  endif()
+  if (CEREAL_FOUND)
+    set(MLPACK_INCLUDE_DIRS ${MLPACK_INCLUDE_DIRS} ${CEREAL_INCLUDE_DIR})
```

Comment:
Ah, I missed this on the initial review!  It seems so obvious to see now.  Anyway, glad to have it fixed :smile: